src/meta-srv/src/region/supervisor.rs

```diff
@@ -259,6 +304,21 @@ impl RegionSupervisor {
         info!("RegionSupervisor is stopped!");
     }
 
+    async fn register_failure_detectors(&self, idents: Vec<Ident>) {
+        let ts_millis = current_time_millis();
+        for ident in idents {
+            // The corresponding region has `acceptable_heartbeat_pause_millis` to send heartbeat from datanode.
+            self.failure_detector
+                .maybe_init_region_failure_detector(ident, ts_millis);
```

Comment:
`maybe_init_region_failure_detector` will insert a heartbeat, but `register_failure_detectors` could be called outside of heartbeat processing path. So there might be a "fake" heartbeat. What's the consequence of doing that?

Reply:
> What's the consequence of doing that?

If the datanode didn't send the heartbeat promptly, the region failover will be trigger again.